packages/site/src/components/LedgerChannelDetails.tsx

```diff
@@ -32,11 +32,23 @@ async function getLedgerDetails(
   };
 
   const lockedBalances = paymentChannels.map((pc) => {
-    const total = pc.Balance.PaidSoFar + pc.Balance.RemainingFunds;
+    let paidSoFar = pc.Balance.PaidSoFar;
+    let remainingFunds = pc.Balance.RemainingFunds;
+
+    if (typeof pc.Balance.RemainingFunds === "string") {
+      remainingFunds = BigInt(parseInt(pc.Balance.RemainingFunds, 16));
+    }
+
+    if (typeof pc.Balance.PaidSoFar === "string") {
```

Comment:
Same comment as above. Maybe we should rebase this PR on https://github.com/statechannels/nitro-gui/pull/68?

Reply:
Okay